unit_tests/test_dockeropts.py

```diff
@@ -0,0 +1,38 @@
+from lib.dockeropts import DockerOpts
+
+
+class TestDockerOpts:
+
```

Comment:
The None and None wont work, but good point, i don't guard against empty string and I expect it will raise a ValueError as dict's cannot have an empty string for a label.